src/anemoi/training/diagnostics/callbacks/__init__.py

```diff
@@ -37,7 +37,13 @@
             plotting.PlotSample,
         ],
     ),
-    ("diagnostics.plot.learned_features", plotting.GraphTrainableFeaturesPlot),
+    (
+        "diagnostics.plot.learned_features",
```

Comment:
The key of `diagnostics.plot.learned_features` has been put in a deprecated dict in the latest changes to the base branch. This change can be incorporated into that. The recommended way to add these plots is through the `callbacks` list in the config

Reply:
Added!